flink-runtime/src/main/java/org/apache/flink/migration/runtime/state/AbstractCloseableHandle.java

```diff
@@ -28,6 +28,8 @@
  * 
  * Offers to register a stream (or other closable object) that close calls are delegated to if
  * the handle is closed or was already closed.
+ *
+ * @deprecated Internal class for savepoint backwards compatability. Don't use for other purposes.
```

Comment:
all right i give up, you go ahead and find all occurrences of "compatability" :D